src/swagger-specs/policy-service.yaml

```diff
@@ -1732,77 +1826,121 @@ definitions:
+                    example: https://platform.adobe.io:443/data/foundation/dulepolicy/enabledCorePolicies
+  EnableCorePolicyViolation:
+    allOf:
+      - $ref: '#/definitions/ViolationsError'
+      - properties:
+          detail:
+            example: Enabling this list of policies will cause violations.
   ViolationsError:
     allOf:
       - $ref: '#/definitions/Error'
+      - properties:
+          status:
+            example: 409
+          title: 
+            example: Violations Error
+          detail:
+            example: The attempted action will cause violations.
       - type: object
-        required:
-          - violations
         properties:
           violations:
             type: array
+            description: A list of violation events, one for each marketing action that resulted in policy violations.
             items:
               $ref: '#/definitions/UpdateViolation'
   UpdateViolation:
     properties:
       enforcementRef:
         type: string
-        example: "/dulepolicy/marketingActions/core/<maRef1>/constraints"
+        description: The attempted marketing action that caused the violation(s) to occur.
+        example: "/dulepolicy/marketingActions/core/exportToThirdParty/constraints"
       entities:
         type: array
+        description: A list of entities that are associated with the violation, including their applied data usage labels.
         items:
           $ref: '#/definitions/Entity'
       violatedPolicies:
         type: array
+        description: A list of policies that were violated as a result of the attempted marketing action.
         items:
           $ref: '#/definitions/PolicyResponse'
   Entity:
-    required:
-      - entityId
     properties:
       name:
         type: string
-        example: "friendly name"
+        description: The name of the entity.
+        example: "Default Merge Policy"
       entityId:
         $ref: '#/definitions/EntityId'
       _etag:
         description: The entity tag (ETag) used to prevent dirty writes.
         type: string
       parents:
         type: array
+        description: A list of entities that have a parent relationship with the current entity.
         items:
           $ref: '#/definitions/EntityId'
       labels:
         type: array
+        description: A list of labels that have been applied to the entity.
         items:
           type: string
           example: "C1"
       options:
         type: array
         items:
           $ref: '#/definitions/OptionName'
-        description: To include all options, use null. To exclude all options use an empty list (i.e. []). To include specific options, list the options to include (i.e. [{option1, option2}])
+        description: An array that allows you to specify which `optionalLabels` To include all options, use null. To exclude all options use an empty list (i.e. []). To include specific options, list the options to include (i.e. [{option1, option2}])
```

Comment:
I think a few words are missing here, since the sentence goes nowhere - I'm guessing you want to say "... to specify which `optionalLabels` to include. To include all options..."?